src/main/java/Record.java

```diff
@@ -1,16 +1,15 @@
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
-import java.util.Date;
 
 public class Record {
     private RecordType type;
-    private String product;
+    private String productName;
     private int count;
     private LocalDate date;
 
     public Record(String inputType, String inputProduct, String inputCount, String inputDate) {
         setType(inputType);
-        product = inputProduct;
+        productName = inputProduct;
         setCount(Integer.parseInt(inputCount));
         setDate(LocalDate.parse(inputDate, DateTimeFormatter.ofPattern("yyyy-MM-dd")));

```

Comment:
I think you can use a DateTimeFormatter constant here - DateTimeFormatter.ISO_LOCAL_DATE

Reply:
Done